demo/style.less

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable */
```

Comment:
disabled this for now. We can run a different PR to run prettier on all of the files if we want.